app/models/agents/google_translation_agent.rb

```diff
@@ -0,0 +1,92 @@
+module Agents
+  class GoogleTranslationAgent < Agent
+    cannot_be_scheduled!
+
+    gem_dependency_check { defined?(Google) && defined?(Google::APIClient) }
```

Comment:
`defined?(Google) &&` is redundant here.  `defined?(Google::APIClient)` won't err if `Google` is undefined.